.github/workflows/go.yml

```diff
@@ -52,12 +52,12 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: ./.github/workflows/go-setup
+# TODO: Convert this
```

Comment:
What is this TODO?

Reply:
hey, sorry it was for me, the tests command i added yesterday, forgor to remove that. Will remove it.